### PR TITLE
CSV generation

### PR DESCRIPTION
**Pull request by Gen0pp**

### CODE REVIEW

File: database/csvGenerator.js [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+      { video: 'https://steamcdn-a.akamaihd.net/steam/apps/256776745/movie480.webm?t=1583175752', thumbnail: 'https://fec-pics.s3.us-east-2.amazonaws.com/FECassets/hlvid3.jpg' },
+      { video: 'https://steamcdn-a.akamaihd.net/steam/apps/256776746/movie480.webm?t=1583175759', thumbnail: 'https://fec-pics.s3.us-east-2.amazonaws.com/FECassets/hlvid4.jpg' }],
+
+    images: ['https://fec-pics.s3.us-east-2.amazonaws.com/FECassets/hl1.jpg', 'https://fec-pics.s3.us-east-2.amazonaws.com/FECassets/hl2.jpg', 'https://fec-pics.s3.us-east-2.amazonaws.com/FECassets/hl3.jpg', 'https://fec-pics.s3.us-east-2.amazonaws.com/FECassets/hl4.jpg', 'https://fec-pics.s3.us-east-2.amazonaws.com/FECassets/hl5.jpg', 'https://fec-pics.s3.us-east-2.amazonaws.com/FECassets/hl6.jpg', 'https://fec-pics.s3.us-east-2.amazonaws.com/FECassets/hl7.jpg', 'https://fec-pics.s3.us-east-2.amazonaws.com/FECassets/hl8.jpg', 'https://fec-pics.s3.us-east-2.amazonaws.com/FECassets/hl9.jpg', 'https://fec-pics.s3.us-east-2.amazonaws.com/FECassets/hl10.jpg', 'https://fec-pics.s3.us-east-2.amazonaws.com/FECassets/hl11.jpg', 'https://fec-pics.s3.us-east-2.amazonaws.com/FECassets/hl12.jpg'],
+  },
+};
```
Comment (ryancmoniz) [inline database/csvGenerator.js:49]: Is this just a featured record or something like that with realistic data?

File: database/csvGenerator.js [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+writer.pipe(fs.createWriteStream('data.csv'));
+
+// CRITERIA
+const options = ['Overwhelmingly Positive', 'Very Positive', 'Positive', 'Mostly Positive', 'Mixed', 'Mostly Negative', 'Negative', 'Very Negative', 'Overwhelmingly Negative'];
+const imgOptions = ['https://fec-pics.s3.us-east-2.amazonaws.com/FECassets/pic1.jpeg', 'https://fec-pics.s3.us-east-2.amazonaws.com/FECassets/pic2.jpg', 'https://fec-pics.s3.us-east-2.amazonaws.com/FECassets/pic3.jpg', 'https://fec-pics.s3.us-east-2.amazonaws.com/FECassets/pic4.jpg', 'https://fec-pics.s3.us-east-2.amazonaws.com/FECassets/pic5.jpg', 'https://fec-pics.s3.us-east-2.amazonaws.com/FECassets/pic6.jpg', 'https://fec-pics.s3.us-east-2.amazonaws.com/FECassets/pic7.jpg', 'https://fec-pics.s3.us-east-2.amazonaws.com/FECassets/pic8.jpg', 'https://fec-pics.s3.us-east-2.amazonaws.com/FECassets/pic9.jpg', 'https://fec-pics.s3.us-east-2.amazonaws.com/FECassets/pic10.jpg', 'https://fec-pics.s3.us-east-2.amazonaws.com/FECassets/pic11.jpg', 'https://fec-pics.s3.us-east-2.amazonaws.com/FECassets/pic12.jpg', 'https://fec-pics.s3.us-east-2.amazonaws.com/FECassets/pic13.jpg', 'https://fec-pics.s3.us-east-2.amazonaws.com/FECassets/pic15.jpg'];
```
Comment (ryancmoniz) [inline database/csvGenerator.js:10]: Are these image points still active and usable?

File: database/csvGenerator.js [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+
+        console.log('drain');
+        writer.once('drain', writeData);
+
+    }
+  }
+
+  writeData();
+
+};
```
Comment (ryancmoniz) [inline database/csvGenerator.js:140]: LGTM!

File: database/csvGenerator.js (hunk 1 above)
Comment (Gen0pp) [reply database/csvGenerator.js:49]: Yeah from the previous author they used as entry 0 before starting their data generation (for presentation I guess) - will go ahead and remove.

File: database/csvGenerator.js (hunk 2 above)
Comment (Gen0pp) [reply database/csvGenerator.js:10]: Unfortunately not... replaced these and regenerated

**Pull request merged**